src/renderer/components/channel-blocker-settings/channel-blocker-settings.vue

```diff
@@ -0,0 +1,56 @@
+<template>
```

Comment:
Please move this to the existing Distraction free settings instead of adding a new column as this is a distraction free feature. Letting it live below the existing toggles should suffice.